src/UserService.Validation/Image/AddImageRequestValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User;
+using LT.DigitalOffice.UserService.Validation.Image.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.UserService.Validation.Image
+{
+  public class AddImageRequestValidator : AbstractValidator<AddImageRequest>, IAddImageRequestValidator
+  {
+    private readonly List<string> imageFormats = new()
+    {
+      ".jpg", ".jpeg", ".png", ".bmp", ".gif", ".tga"
+    };
+
+    public AddImageRequestValidator()
+    {
+      RuleFor(x => x.Content)
+        .NotEmpty().WithMessage("Content can't be empty")
```

Comment:
need fix format code

Reply:
i think it's ok
2 spaces before .NotEmpty, WithMessage in one line with not empty, because it is not too long

Reply:
we write all under each other